src/main/config/config_streamer.c

```diff
@@ -53,6 +53,8 @@ extern uint8_t __config_end;
 #  define FLASH_PAGE_SIZE                 ((uint32_t)0x8000)
 # elif defined(UNIT_TEST)
 #  define FLASH_PAGE_SIZE                 (0x400)
+# elif defined(SITL)
```

Comment:
I think `SITL` is not really understandable to somebody who is working on a change not specifically targeting the SITL build. We should try and come up with a more self-explaining name for this define (`SIMULATOR_BUILD`?).